control/activate_venv.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+CCHQ_VIRTUALENV=${CCHQ_VIRTUALENV:-cchq}
+VENV=~/.virtualenvs/$CCHQ_VIRTUALENV
+quiet_mode='false'
+
+while test $# -gt 0; do
+  case "$1" in
+    -q|--quiet)
+      quiet_mode='true'
+      shift
+      ;;
+    *)
+      break
+      ;;
+  esac
+done
+
+if [[ $BIONIC_USE_SYSTEM_PYTHON == true ]]; then
+    echo "The variable BIONIC_USE_SYSTEM_PYTHON is set in your environment."
+    echo "This variable should only be used temporarily when it is absolutely necessary to use Python 3.6 on 18.04."
+    echo "Please remove this variable from your environment when you are able to use Python 3.10 again."
+fi
+
+# if on 18.04 with 3.10 installed, use cchq-3.10 unless $BIONIC_USE_SYSTEM_PYTHON is true
+if [[ $BIONIC_USE_SYSTEM_PYTHON == false ]] && hash python3.10 2>/dev/null && [[ $( source /etc/os-release; echo $VERSION_ID ) == 18.04 ]]; then
+    # only append 3.10 if it is not already in the name
+    if [[ $CCHQ_VIRTUALENV != *"3.10"* ]]; then
+        CCHQ_VENV_PATH_OLD=$VENV
+        CCHQ_VIRTUALENV=$CCHQ_VIRTUALENV-3.10
+    fi
+    VENV=~/.virtualenvs/$CCHQ_VIRTUALENV
+fi
+
+# activate virtualenv if it exists, otherwise exit with error
+if [ -f $VENV/bin/activate ]; then
+    source $VENV/bin/activate
+    exit 0
+else
+    if [ "$quiet_mode" = "false" ] then;
```

Comment:
Two things going on here:

1. The previous semicolon position is a syntax error.
2. **Suggestion**: when using an internal "bool" var (whose value you know is either `true` or `false` because your script controls it), it's simpler to just execute the value of the var because the `true` and `false` binaries are always present on all POSIX-compliant OSes.


```suggestion
    if $quiet_mode; then
```

Reply:
> The previous semicolon position is a syntax error.

Distinctly remember writing this and thinking to myself "that's not right" but I clearly did not go back to check 😅 

@skodde and I had [a discussion](https://github.com/dimagi/commcare-cloud/pull/5574#discussion_r1011013702) about bool string values in bash not too long ago. I think you may have differing opinions and my goal is not to ruffle feathers, but to see if we can come to a consensus for consistency's sake. If we chalk it up to personal preference that is also fine!

I do like the idea of `if $quiet_mode; then` in cases where we are absolutely sure the variable will only be "true" or "false", but my question is are we able to guarantee that? Based on my understanding, I'd assume yes but maybe that's wrong.

Reply:
Just realized the thread I linked doesn't discuss bools, so I totally made that. It makes sense why we would want to wrap the content of non-bools in quotations since we cannot make assumptions about its value. Sorry for the unnecessary ping!